srcpkgs/python3-qmk/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'python3-qmk'
+pkgname=python3-qmk
+version=0.0.27
+revision=1
+archs=noarch
+create_wrksrc=yes
+build_wrksrc="./qmk-${version}"
+build_style=python3-module
+hostmakedepends="unzip python3-setuptools"
+depends="python3-appdirs python3-argcomplete python3-colorama python3-hjson python3-nose2 flake8 python3-yapf dfu-programmer avrdude dfu-util avr-gcc cross-arm-none-eabi-gcc"
```

Comment:
Wrap this at 80 characters.

Reply:
Done, the linter didn't warn me but indeed much cleaner.